osbs/build/build_requestv2.py

```diff
@@ -600,6 +600,25 @@ def __init__(self, build_json_store, outer_template=None):
     def render(self, validate=True):
         return super(SourceBuildRequest, self).render(validate=validate)
 
+    def render_name(self, name, image_tag, platform):
+        """Sets the Build/BuildConfig object name
+
+        Source container builds must have unique names, because we are not
+        using buildConfigs just regular builds
+
+        platform is unused for source container images builds
+        """
+        name = image_tag
+
+        _, salt, timestamp = name.rsplit('-', 2)
+
+        name = '{}-{}'.format(salt, timestamp)
```

Comment:
TODO: it misses name, current builds are just:
* `90336-20191107115102`
* `97155-20191107115112`

Reply:
It seems that we don't care about names for builds without stored buildconfig. I just added `sources-` prefix